### PR TITLE
Don't count afk players CFC-66

### PR DESCRIPTION
**Pull request by wrefgtzweve**
Uses hooks from https://github.com/brandonsturgeon/cfc_antiafk_custom/pull/14

### CODE REVIEW

File: lua/cfc_dynamic_limits/sv_checkup.lua [hunk 1]
```diff
@@ -32,9 +32,20 @@ local function checkModule( plyCount, mod )
     end
 end
 
+local function shouldCountPlayer( ply )
+    if ply:GetNWBool( "CFC_AntiAFK_IsAFK" ) then return false end -- Anti-AFK
+
+    return true
+end
```
Comment (sarahsturgeon) [inline lua/cfc_dynamic_limits/sv_checkup.lua]:
```suggestion
local function shouldCountPlayer( ply )
    return ply:GetNWBool( "CFC_AntiAFK_IsAFK", false )
end
```

**Review by sarahsturgeon — APPROVED** [review]
Maybe our `shouldCountPlayer` thing should run a hook that AntiAFK hooks into instead?

Any ideas on the pros/cons of each?

File: lua/cfc_dynamic_limits/sv_checkup.lua (hunk 1 above)
Comment (wrefgtzweve) [reply lua/cfc_dynamic_limits/sv_checkup.lua]: Hook is better yeah

**Review by sarahsturgeon — APPROVED** [review]
👍

**Pull request merged**